examples/window.rs

```diff
@@ -160,7 +172,23 @@ impl Application {
         Ok(window_id)
     }
 
-    fn handle_action(&mut self, event_loop: &ActiveEventLoop, window_id: WindowId, action: Action) {
+    fn handle_action(&mut self, event_loop: &ActiveEventLoop, action: Action) {
+        match action {
+            Action::PrintHelp => self.print_help(),
```

Comment:
I'm pretty sure it won't work here, since it's triggered from the user input.

I'd generally create a separate channel to handle dumping monitors/wakeup on web, and make it web specific if you really need those channels.

Reply:
> I'm pretty sure it won't work here, since it's triggered from the user input.

How is this related to user input?
I can confirm it does actually work.

> I'd generally create a separate channel to handle dumping monitors/wakeup on web, and make it web specific if you really need those channels.

This already has a use-case on native:
I added triggering a proxy wake up with "S".

Seeing your other comments I think you should just try this yourself to see how it works and what it does.

Reply:
>This already has a use-case on native:
I added triggering a proxy wake up with "S".

In this case, could you split the actions properly?

Reply:
Done, I also remove the `_ => ...` shortcut and renamed the function, so the flow should be easier to understand now.